src/pages/ResetPassword/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Container, FormContainer } from './style';
+import { useLocation } from 'react-router-dom';
+import { api, history } from '../../services';
+import { toast } from 'react-toastify';
+
+function useQuery() {
+    return new URLSearchParams(useLocation().search);
+}
+
+const ResetPassword = () => {
+    const query = useQuery();
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [resetErrors, setResetErrors] = useState({});
+
+    async function postResetPassword(e) {
+        e.preventDefault();
+
+        try {
+            await api.post('/api/password/reset', {
+                email: email,
+                password: password,
+                token: query.get('token'),
+            });
+            history.push('/');
+        } catch (error) {
+            const { message, errors } = error.response.data;
+            setResetErrors({
+                ...resetErrors,
+                ...errors,
+            });
+            toast.error(message);
+        }
+    }
+
+    return (
+        <Container>
+            <FormContainer>
+                <form className="form">
+                    <div>
+                        <div className="form__group form__group--email">
+                            <label htmlFor="email">EMAIL</label>
+                            <input
+                                className={Object.keys(resetErrors).length === 0 ? 'gray' : 'red'}
```

Comment:
Você não pode dizer que todos campos são invalidos, se tiver apenas um erro, tem que ser o erro naquele campo.
 ```suggestion
                                className={!resetErrors.email ? 'gray' : 'red'}
```